src/GUI/UI/UIPopup.cpp

```diff
@@ -262,7 +264,9 @@ void GameUI_DrawItemInfo(struct ItemGen *inspect_item) {
         }
         inspect_item->UpdateTempBonus(pParty->GetPlayingTime());
         if (inspect_item->IsBroken()) {
-            if (pPlayers[pParty->getActiveCharacter()]->CanRepair(inspect_item) == 1)
+            if (pParty->getOptionallySharedSkillStrongestEffect(
+                    PLAYER_SKILL_TYPE::PLAYER_SKILL_REPAIR,
```

Comment:
Please add TODO to add changing the reaction origin to the character who's skill was used.